ChangeLog

```diff
@@ -1,3 +1,75 @@
+* Thu Oct 15 2020 Chris Lumens <clumens@redhat.com> Pacemaker-2.0.5
+- Changesets: 497
+- Diff: 285 files changed, 22704 insertions(+), 14458 deletions(-)
+
+- Features added since Pacemaker-2.0.4
+  + cibsecret: read value to set from input (or stdin) if not specified
+  + controller: add new IPC API command for getting node list
```

Comment:
API isn't considered features of user interest, so we don't put them in the changelog.

However I have been thinking we should start documenting public API changes in a release, probably a new section just before "Changes". I wouldn't bother this time unless you want to go through all the public API changes since 2.0.4 (per your soname changes, libcrmcommon, libpe_status and libpe_rules changed).

Reply:
2.1.0 seems like a great time to start doing this.